torch/_inductor/codegen/triton.py

```diff
@@ -565,6 +553,47 @@ def ceil(x):
         return f"tl.math.ceil({x})"
 
 
+class TritonKernelOverrides(TritonOverrides):
```

Comment:
Leave a comment here or in TritonOverrides explaining when to use one or the other.